deployments/ecs/ec2/README.md

```diff
@@ -0,0 +1,156 @@
+
+### Launch the Collector
+The Collector is designed to be run as a Daemon service in an EC2 ECS cluster.
+
+To create a Collector service from the Amazon ECS console:
+
+Go to your cluster in the console
+1. Click on the "Services" tab.
+2. Click "Create" at the top of the tab.
+3. Select:
+   - Launch Type -> EC2
+   - Task Definition (Family) -> splunk-otel-collector
+   - Task Definition (Revision) -> 1 (or whatever the latest is in your case)
+   - Service Name -> splunk-otel-collector
+   - Service type -> DAEMON
+4. Leave everything else at default and click "Next step"
+5. Leave everything on this next page at their defaults and click "Next step". 
+6. Leave everything on this next page at their defaults and click "Next step". 
+7. Click "Create Service" and the agent should be deployed onto each node in the ECS cluster. You should see infrastructure and docker metrics flowing soon.
+
+## Custom Configuration
+To use a custom configuration file, replace the value of environment variable
+`SPLUNK_CONFIG` with the file path of the custom configuration file in Collector
+task definition.
+
+Alternatively, you can specify the custom configuration YAML directly using environment
+variable `SPLUNK_CONFIG_YAML` as describe [below](#direct-configuration).
+
+### ecs_observer
+Use extension
+[Amazon Elastic Container Service Observer](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/observer/ecsobserver#amazon-elastic-container-service-observer)
+(`ecs_observer`) in your custom configuration to discover metrics targets
+in running tasks, filtered by service names, task definitions and container labels.
+`ecs_observer` is currently limited to Prometheus targets and requires the read-only
+permissions below. You can add the permissions to the task role by adding them to a 
+customer-managed policy that is attached to the task role.
+```text
+ecs:List*
+ecs:Describe*
+```
+
+Below is an example of a custom configuration in which the `ecs_observer` is configured to find
+Prometheus targets in cluster `lorem-ipsum-cluster`, region `us-west-2`, where the task ARN
+pattern is `^arn:aws:ecs:us-west-2:906383545488:task-definition/lorem-ipsum-task:[0-9]+$`.
+The results are written to file `/etc/ecs_sd_targets.yaml`. The `prometheus` receiver is
+configured to read targets from the results file. The values for `access_token`
+and `realm` are read from environment variables `SPLUNK_ACCESS_TOKEN` and `SPLUNK_REALM`
+respectively, which must be specified in your container definition.
+
+```yaml
+extensions:
+  ecs_observer:
+    refresh_interval: 10s
+    cluster_name: 'lorem-ipsum-cluster'
+    cluster_region: 'us-west-2'
+    result_file: '/etc/ecs_sd_targets.yaml'
+    task_definitions:
+      - arn_pattern: "^arn:aws:ecs:us-west-2:906383545488:task-definition/lorem-ipsum-task:[0-9]+$"
+        metrics_ports: [9113]
+        metrics_path: /metrics
+receivers:
+  signalfx:
```

Comment:
nit: this is configured, but not enabled in the pipeline

Reply:
Yes, I removed it.